tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +148,21 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert set(tai.inventory) == set([item_a, item_b, item_f])
+    assert set(jesse.inventory) == set([item_c, item_d, item_e])
```

Comment:
I'm not 100% sure why y'all chose to cast these lists to a set, but I believe it is because the way the lists are currently set up does not match what we would see  as a result of the swap. In that case, rather than casting to a set, make sure you have a strong understanding of how your lists behave as the result of a swap!